DCAF.Core.lua

```diff
@@ -7901,9 +7930,15 @@ local function buildTankerMenus(caption, scope)
 end
 rebuildTankerMenus = buildTankerMenus
 
+ --- Build AAR menus for each player group when player enters plane. Recommended to use with
+  -- default parameters.
+  -- @param #string caption The title of the menu. Default is "AAR"
+  -- @param scope DEPRECATED
 function DCAF.TankerTracks:BuildF10Menus(caption, scope)

```

Comment:
If you remove the `buildTankerMenus()`call and instead only uses the `OnPlayerEnteredAirplane` event to build F10 menus that are scoped to the spawned guy, our controllers will no longer be able to control tankers. Also, you cannot scope menus to individual UNITs. When you do that, the `buildTankerMenus` function will resolve it to the UNIT's GROUP, which means a menu will be built for the entire group every time a new member slots in, resulting in multiple menus (or just corrupt state)